server/db/models/user.js

```diff
@@ -18,12 +18,15 @@ const User = db.define('user', {
   githubId: {
     type: Sequelize.INTEGER
   },
-  isAdmin:{
-    type: Sequelize.BOOLEAN
-  },
-  isAuth:{
-    type: Sequelize.BOOLEAN
-  }
+  // name: {
```

Comment:
Remove all model changes, I will update the models to have default values and handle null or empty string edge cases